servicebus/data-plane/azure-servicebus/src/main/java/com/microsoft/azure/servicebus/management/TopicDescriptionSerializer.java

```diff
@@ -222,6 +220,8 @@ private static TopicDescription parseFromEntry(Node xEntry) {
                                     case "SupportOrdering":
                                         td.supportOrdering = Boolean.parseBoolean(element.getFirstChild().getNodeValue());
                                         break;
+                                    default:
```

Comment:
[SpotBugs-P2] Switch statement found in com.microsoft.azure.servicebus.management.TopicDescriptionSerializer.parseFromEntry(Node) where default case is missing